### ISSUE TITLE
Package request: Acter

### ISSUE THREAD

**Issue by Thatoo** [NixOS/nixpkgs#352997]
**Project description**

Your social organizing app build on [matrix](https://matrix.org/): think WhatsApp, but with proper task lists, structured discussion, etc.

**Metadata**

* homepage URL: https://acter.global/
* source URL: https://github.com/acterglobal/a3
* license:  agplv3
* platforms: unix, linux, darwin, ...

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by pluiedev** [comment]
Just a small correction - Acter is not GPLv3 but uses its own custom (non-FOSS) license that would turn into AGPLv3 after a given date. I'm not sure how to model this in Nixpkgs but it's pretty similar to MariaDB's Business Source License which I believe we already have here.

**Comment by Thatoo** [comment]
Yes, I made a mistake typing, I missed the A of AGPL, sorry. That's what I found https://github.com/acterglobal/a3/blob/main/LICENSE-AGPL.txt. I didn't know about a custom non-free license, sorry.
